crds/operators.coreos.com_catalogsources.yaml

```diff
@@ -90,6 +90,11 @@ spec:
                   a registry-server with. Only used when SourceType = SourceTypeGrpc.
                   If present, the address field is ignored.
                 type: string
+              priority:
+                description: Priority field, if specified, is used to rank catalog
```

Comment:
Is a catalog source with priority 1 preferred to a catalog source with priority 2? Or -1?

Reply:
right, we are defining that in the OLM logic. Here is just a type. Personally don't think we need to call it out here right?

Reply:
As a consumer of an API, I would like to know what effect changes to this field will have. You may also want to check out the [CRD validation markers](https://book.kubebuilder.io/reference/markers/crd-validation.html), to see how we can  bound this fields and avoid headaches parsing bad values ourselves.

Reply:
The semantics should be unambiguous here. The API shouldn't define behaviors of its consumers, but it must clearly define both structure AND meaning. Otherwise, one consumer can decide that lower integers rank first, and another can decide that higher integers rank first, and neither one is wrong, because as far as the API is concerned, it's only a number divorced from meaning.